language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/Template.java

```diff
@@ -4,10 +4,11 @@
 package com.daml.ledger.javaapi.data;
 
 import com.daml.ledger.javaapi.data.codegen.CreateAnd;
+import com.daml.ledger.javaapi.data.codegen.Update;
 
 public abstract class Template extends com.daml.ledger.javaapi.data.codegen.DamlRecord<Template> {
 
-  public abstract CreateCommand create();
+  public abstract Update<?> create();
```

Comment:
This can return `Update<? extends ContractId<? extends Template>>`.

Alternatively, consider adding a type parameter `T` to `Template`, in which case `create` returns `Update<? extends ContractId<T>>`.